src/wallet/rpcwallet.cpp

```diff
@@ -41,6 +41,8 @@
 
 #include <functional>
 
+#include <boost/optional.hpp>
```

Comment:
I'd mildly prefer to redirect this through our `optional.h`, define our own `make_optional` like the `Option<T>` itself instead of boost appearing here direclty.